pkg/api/handler.go

```diff
@@ -0,0 +1,7 @@
+package api
+
+import "github.com/gin-gonic/gin"
+
+type handler interface {
```

Comment:
I will like to talk to you about it because it seems to me that there isn't interface declaration in the atlas-profile and it's a bit weird. anyway I changed the name